### PR TITLE
Dbfile

### PR DESCRIPTION
**Pull request by mikeclimbrock**

### CODE REVIEW

File: api/app/controllers/Helpers.scala [hunk 1]
```diff
@@ -3,11 +3,15 @@ package controllers
...
-import play.Play
 
 object Helpers {
-  val dbFilePath = Play.application().getFile("conf/GeoLite2-City.mmdb").getAbsolutePath
-  val geoIp = MaxMindIpGeo(dbFilePath, 1000)
+  import sys.process._
+  import java.net.URL
+  import java.io.File
+  import scala.language.postfixOps
+  new URL("https://s3.amazonaws.com/io.flow.aws-s3-public/location/GeoLite2-City.mmdb") #> new File("GeoLite2-City.mmdb") !!
```
Comment (paololim) [inline api/app/controllers/Helpers.scala]: Should we use the s3 api?

Comment (mikeclimbrock) [inline api/app/controllers/Helpers.scala]: was using this:
http://alvinalexander.com/scala/scala-how-to-download-url-contents-to-string-file

Comment (mikeclimbrock) [inline api/app/controllers/Helpers.scala]: I'd prefer to leave the dependency out for now, one less thing to manage

Comment (paololim) [inline api/app/controllers/Helpers.scala]: Yeah, that's fine. Will that db always be public, or something we'll make private later when we purchase? We can figure out the extra dependency later, too.

Comment (mikeclimbrock) [inline api/app/controllers/Helpers.scala]: think we should punt for now, in theory, would be cool to keep it public

**Comment by mbryzek** [comment]
Looks great. I think we will eventually want to make the URL come from configuration - then by default it can use this public data set, but when we buy the better quality data set we can easily load that.

**Pull request merged**